fe/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -683,6 +684,7 @@ private void handleInsertStmt() throws Exception {
                 return;
             }
 
+            MetricRepo.COUNTER_LOAD_FINISHED.increase(1L);
```

Comment:
What is the `COUNTER_LOAD_FINISHED` means? If it means that load is done successfully, 
this counter should be added after `commitAndPublishTransaction()`